core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -813,14 +813,6 @@ void core_init_mmu(struct tee_mmap_region *mm)
 	assert(max_va < BIT64(CFG_LPAE_ADDR_SPACE_BITS));
 }
 
-bool core_mmu_place_tee_ram_at_top(paddr_t paddr)
-{
-	size_t base_level_size = BASE_XLAT_BLOCK_SIZE;
-	paddr_t base_level_mask = base_level_size - 1;
-
-	return (paddr & base_level_mask) > (base_level_size / 2);
```

Comment:
With this we were trying to avoid crossing boundaries of the top level tables, typically 1G on Armv7.
Crossing such a boundary costs 4k since we need to use one more translation table.

Perhaps this small memory saving isn't relevant on LPAE enabled platforms.